resources/js/dashboard/pages/administrator/users/components/Create.vue

```diff
@@ -179,7 +194,7 @@ export default {
       if (this.btnTitle === "Enreigistrer") {
         if (
           this.form.password === this.form.confirmPassword &&
-          this.form.roles.length !== 0
+          this.form.roles.length !== 0 && this.form.hospitals.length !== 0
```

Comment:
Si c'est un administrateur, il ne sera pas forcement lié à un hopital
delete `&& this.form.hospitals.length !== 0`